<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>OCR A Extended</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Contact Book</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/icon/Icon.ico</normalon>
   </iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="iconSize">
   <size>
    <width>26</width>
    <height>26</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_top">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>45</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(55, 55, 20);
color: rgb(255, 207, 0);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_new">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>OCR A Extended</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	margin-left: 10px;
	background-color: rgb(100, 100, 100);
	border-radius: 5px;
	border-left: none;
	border-right: 2px solid rgb(0, 0, 0);
	border-bottom: 2px solid rgb(0, 0, 0);
}
QPushButton:hover {
	background-color: rgb(90, 90, 90);
}
QPushButton:pressed {
	background-color: rgb(255, 207, 0);
	border-right: 2px solid rgb(255, 120, 0);
	border-bottom: 2px solid rgb(255, 120, 0);
	color: rgb(35, 35, 35);
}</string>
         </property>
         <property name="text">
          <string>New</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_delete">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>OCR A Extended</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	margin-left: 10px;
	background-color: rgb(100, 100, 100);
	border-radius: 5px;
	border-left: none;
	border-right: 2px solid rgb(0, 0, 0);
	border-bottom: 2px solid rgb(0, 0, 0);
}
QPushButton:hover {
	background-color: rgb(90, 90, 90);
}
QPushButton:pressed {
	background-color: rgb(255, 207, 0);
	border-right: 2px solid rgb(255, 120, 0);
	border-bottom: 2px solid rgb(255, 120, 0);
	color: rgb(35, 35, 35);
}</string>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_delete_all">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>OCR A Extended</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	margin-right: 10px;
	background-color: rgb(100, 100, 100);
	border-radius: 5px;
	border-left: none;
	border-right: 2px solid rgb(0, 0, 0);
	border-bottom: 2px solid rgb(0, 0, 0);
}
QPushButton:hover {
	background-color: rgb(90, 90, 90);
}
QPushButton:pressed {
	background-color: rgb(255, 207, 0);
	border-right: 2px solid rgb(255, 120, 0);
	border-bottom: 2px solid rgb(255, 120, 0);
	color: rgb(35, 35, 35);
}</string>
         </property>
         <property name="text">
          <string>Delete All</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_out">
         <property name="minimumSize">
          <size>
           <width>55</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>OCR A Extended</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	margin-right: 10px;
	background-color: rgb(100, 100, 100);
	border-radius: 20.3px;
	border:2px solid rgb(35, 35, 35)
}
QPushButton:hover {
	background-color: rgb(90, 90, 90);
}
QPushButton:pressed {
	background-color: rgb(255, 207, 0);
	border: 2px solid rgb(255, 120, 0);
	color: rgb(35, 35, 35);
}</string>
         </property>
         <property name="text">
          <string>Out</string>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_center">
      <property name="minimumSize">
       <size>
        <width>700</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(41, 41, 59);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_header">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>65</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image: url(:/header/top_bar2.png);
background-position: center;
background-repeat: no-repeat;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="tableView">
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>950</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>OCR A Extended</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QTableView {
	color: rgb(225, 225, 255);
	border: 1px solid rgb(255, 207, 0);
	background-color: rgb(41, 41, 59);

}
QTableView:hover {
	
	background-color: rgb(41, 41, 55);
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_bottom">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>45</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/background_bottom/Images/background_bottom.png);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image: url(:/footer/background_bottom.png);
color: rgb(255, 207, 0);

</string>
         </property>
         <property name="text">
          <string>Designed by Miguel Victor</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../resources/resources_main.qrc"/>
 </resources>
 <connections/>
</ui>
